android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java

```diff
@@ -84,6 +87,10 @@ public PlayerView createViewInstance(ThemedReactContext context) {
     player = Player.create(reactContext, playerConfig);
 
     playerView = new PlayerView(reactContext, player);
+/*     subtitleView = new SubtitleView(reactContext, null);
```

Comment:
Unless this is a WIP PR, I don't think we should commit commented-out code

Reply:
it is a WIP, about subtitles. I will remove it later.

Reply:
OK, thanks, I've updated the PR title and converted it to a draft, to make this clear.